src/plugins/data/server/ui_settings.ts

```diff
@@ -747,5 +747,15 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       schema: schema.boolean(),
     },
+    [UI_SETTINGS.POLLING_INTERVAL]: {
+      name: i18n.translate('data.advancedSettings.query.dataSourceReadOnlyTitle', {
```

Comment:
fast follow: make this namespace like "data.advancedSettings.query.pollingInterval"